2023-11-30 22:41:08.555 [info] 




2023-11-30 22:41:08.578 [info] Extension host agent started.
2023-11-30 22:41:08.739 [info] [127.0.0.1][7f35c3e0][ManagementConnection] New connection established.
2023-11-30 22:41:08.800 [info] [127.0.0.1][18f172fd][ExtensionHostConnection] New connection established.
2023-11-30 22:41:08.933 [info] [127.0.0.1][18f172fd][ExtensionHostConnection] <16165> Launched Extension Host Process.
2023-11-30 22:41:11.249 [error] CodeExpectedError: Could not find pty on pty host
    at $.P (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:32310)
    at $.updateTitle (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:27490)
    at T.<computed> (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:22750)
    at Object.call (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:17:4907)
    at u.s (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:5150)
    at u.q (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4661)
    at i.value (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4066)
    at C.w (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:1902)
    at C.fire (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:2119)
    at process.G (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:8:27561)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2023-11-30 22:41:13.038 [info] [127.0.0.1][7f35c3e0][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
2023-11-30 22:41:13.039 [error] Error: Throttler is disposed
    at c.queue (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:13632)
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:15216
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:14852
2023-11-30 22:41:16.207 [info] Got delay-shutdown request while in shutdown timeout, delaying
2023-11-30 22:41:16.207 [info] Cancelling previous shutdown timeout
2023-11-30 22:41:16.768 [info] [127.0.0.1][5c8e58e4][ManagementConnection] New connection established.
2023-11-30 22:41:16.788 [info] [127.0.0.1][e6727f79][ExtensionHostConnection] New connection established.
2023-11-30 22:41:16.796 [info] [127.0.0.1][e6727f79][ExtensionHostConnection] <16295> Launched Extension Host Process.
2023-11-30 22:41:18.057 [info] [127.0.0.1][18f172fd][ExtensionHostConnection] <16165> Extension Host Process exited with code: 0, signal: null.
2023-11-30 22:41:18.057 [info] Cancelling previous shutdown timeout
2023-11-30 22:41:19.210 [error] [File Watcher (parcel)] Unexpected error: inotify_add_watch on '/home/tianyi/.vscode-server/data/logs/20231121T065951/exthost3/vscode.github' failed: No such file or directory (EUNKNOWN) (path: /home/tianyi)
2023-11-30 22:41:19.210 [error] [File Watcher (universal)] restarting watcher after error: inotify_add_watch on '/home/tianyi/.vscode-server/data/logs/20231121T065951/exthost3/vscode.github' failed: No such file or directory
2023-11-30 22:42:42.745 [warning] No ptyHost heartbeat after 6 seconds
2023-11-30 23:41:55.333 [info] [127.0.0.1][5c8e58e4][ManagementConnection] The client has reconnected.
2023-11-30 23:41:55.540 [info] [127.0.0.1][e6727f79][ExtensionHostConnection] The client has reconnected.
2023-11-30 23:41:56.381 [error] CodeExpectedError: Could not find pty on pty host
    at $.P (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:32310)
    at $.updateIcon (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:27540)
    at T.<computed> (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:22750)
    at Object.call (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:17:4907)
    at u.s (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:5150)
    at u.q (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4661)
    at i.value (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4066)
    at C.w (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:1902)
    at C.fire (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:2119)
    at process.G (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:8:27561)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2023-11-30 23:41:56.383 [error] CodeExpectedError: Could not find pty on pty host
    at $.P (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:32310)
    at $.updateTitle (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:27490)
    at T.<computed> (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:22750)
    at Object.call (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:17:4907)
    at u.s (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:5150)
    at u.q (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4661)
    at i.value (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4066)
    at C.w (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:1902)
    at C.fire (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:2119)
    at process.G (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:8:27561)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2023-12-01 00:10:43.769 [info] [127.0.0.1][5c8e58e4][ManagementConnection] The client has reconnected.
2023-12-01 00:11:10.626 [info] [127.0.0.1][e6727f79][ExtensionHostConnection] The client has reconnected.
2023-12-01 00:17:15.899 [info] [127.0.0.1][2838c01a][ManagementConnection] New connection established.
2023-12-01 00:17:16.127 [info] [127.0.0.1][5d51e49a][ExtensionHostConnection] New connection established.
2023-12-01 00:17:16.142 [info] [127.0.0.1][5d51e49a][ExtensionHostConnection] <40074> Launched Extension Host Process.
2023-12-01 00:17:22.931 [error] CodeExpectedError: Could not find pty on pty host
    at $.P (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:32310)
    at $.updateIcon (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:27540)
    at T.<computed> (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:22750)
    at Object.call (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:17:4907)
    at u.s (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:5150)
    at u.q (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4661)
    at i.value (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4066)
    at C.w (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:1902)
    at C.fire (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:2119)
    at process.G (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:8:27561)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2023-12-01 00:17:22.933 [error] CodeExpectedError: Could not find pty on pty host
    at $.P (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:32310)
    at $.updateTitle (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:27490)
    at T.<computed> (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:27:22750)
    at Object.call (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:17:4907)
    at u.s (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:5150)
    at u.q (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4661)
    at i.value (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:15:4066)
    at C.w (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:1902)
    at C.fire (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:10:2119)
    at process.G (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/platform/terminal/node/ptyHostMain.js:8:27561)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2023-12-01 00:19:59.606 [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /usr/local/lib/python3.9/dist-packages)
2023-12-01 00:22:21.063 [info] [127.0.0.1][e6727f79][ExtensionHostConnection] <16295> Extension Host Process exited with code: 0, signal: null.
2023-12-01 02:04:53.881 [info] [127.0.0.1][69d8495a][ManagementConnection] New connection established.
2023-12-01 02:04:53.883 [info] [127.0.0.1][7e5a6e32][ExtensionHostConnection] New connection established.
2023-12-01 02:04:53.907 [info] [127.0.0.1][7e5a6e32][ExtensionHostConnection] <49770> Launched Extension Host Process.
2023-12-01 02:04:57.473 [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /home/tianyi)
2023-12-01 02:04:57.515 [info] [127.0.0.1][69d8495a][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
2023-12-01 02:04:57.516 [error] Error: Throttler is disposed
    at c.queue (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:13632)
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:15216
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:14852
    at runNextTicks (node:internal/process/task_queues:60:5)
    at listOnTimeout (node:internal/timers:538:9)
    at process.processTimers (node:internal/timers:512:7)
2023-12-01 02:04:57.519 [error] Error: Throttler is disposed
    at c.queue (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:13632)
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:15216
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:14852
    at runNextTicks (node:internal/process/task_queues:60:5)
    at listOnTimeout (node:internal/timers:538:9)
    at process.processTimers (node:internal/timers:512:7)
2023-12-01 02:04:57.519 [error] Error: Throttler is disposed
    at c.queue (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:13632)
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:15216
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:14852
    at runNextTicks (node:internal/process/task_queues:60:5)
    at listOnTimeout (node:internal/timers:538:9)
    at process.processTimers (node:internal/timers:512:7)
2023-12-01 02:04:57.519 [error] Error: Throttler is disposed
    at c.queue (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:13632)
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:15216
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:14852
    at runNextTicks (node:internal/process/task_queues:60:5)
    at listOnTimeout (node:internal/timers:538:9)
    at process.processTimers (node:internal/timers:512:7)
2023-12-01 02:04:57.519 [error] Error: Throttler is disposed
    at c.queue (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:13632)
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:15216
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:14852
    at runNextTicks (node:internal/process/task_queues:60:5)
    at listOnTimeout (node:internal/timers:538:9)
    at process.processTimers (node:internal/timers:512:7)
2023-12-01 02:04:57.519 [error] Error: Throttler is disposed
    at c.queue (/home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:13632)
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:15216
    at /home/tianyi/.vscode-server/bin/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/out/vs/server/node/server.main.js:70:14852
    at runNextTicks (node:internal/process/task_queues:60:5)
    at listOnTimeout (node:internal/timers:538:9)
    at process.processTimers (node:internal/timers:512:7)
2023-12-01 02:09:59.023 [info] [127.0.0.1][5d51e49a][ExtensionHostConnection] <40074> Extension Host Process exited with code: 0, signal: null.
